lib/player.rb

```diff
@@ -26,6 +26,19 @@ def reflect_state(str)
     @state.reflect_state(str, @token)
   end
 
+  def fetch_play(_)
+    print "\nPlayer #{@token}, your move: "
+    gets.chomp.to_i
```

Comment:
#### `REQUIRED`

Let's move UI away from game logic.

> `main.rb` file should be responsible for user interaction. This is the only one place where `puts` and `get.chomp` can be used.

Reply:
Hi Bola,

Regarding the README, the instructions are already there in the branch we want to merge (build-ai). I think they are clear, but if you want to suggest some clarification, we're open to it.

Regarding the 'gets' in Player.fetch_play, it's there precisely because of the single responsibility rule, 'cause each player is responsible for calculating his next move, and the human player's way of "calculating" that is by asking the human. But I have some ideas on how to achieve that separation and single responsibility at the same time, so I'll try that; I just wanted to state that that 'gets' has a reason to be there. :-) 

And regarding validation. we did a lousy job (or no job at all) :-(  We'll fix that ASAP; your remarks are great, thanks!